test/api_api_tokens_test.go

```diff
@@ -26,14 +26,13 @@ func Test_client_APITokensAPIService(t *testing.T) {
 		expireDate, err := time.Parse(time.RFC3339, "2021-01-01T00:00:00.000Z")
 		require.Nil(t, err)
 		environment := "development"
-		token := client.CreateApiTokenSchemaOneOf2{
+		token := client.CreateApiTokenSchema{
```

Comment:
This is a nice improvement. I'm guessing our API changed. Does this still work with 6.x now?

Reply:
It's the API changing or just the new generator being a bit smarter.